### PR TITLE
iCj

### PR DESCRIPTION
**Pull request by wargio**
Ref https://github.com/radare/radare2/pull/7428

### CODE REVIEW

File: t.formats/pe/signature_cert [hunk 1]
```diff
@@ -67,3 +68,13 @@ EXPECT='signed   true
 '
 IGNORE_ERR=1
 run_test
+
+
+NAME='PE: corkami signature.exe - iCj'
+FILE=../../bins/pe/signature.exe
+ARGS=''
+CMDS="iCj; q!"
```
Comment (radare) [inline t.formats/pe/signature_cert]: what about using ~{} ?

q! is not necessary. its implicit

File: t.formats/pe/signature_cert [hunk 2]
```diff
@@ -54,6 +54,7 @@ EXPECT='signedData
       Digest Encryption Algorithm
         rsaEncryption
       Unauthenticated Attributes:
+
```
Comment (radare) [inline t.formats/pe/signature_cert]: can we get rid of this unnecesary empty line?

Comment (wargio) [inline t.formats/pe/signature_cert]: yes

File: t.formats/pe/signature_cert (hunk 1 above)
Comment (wargio) [inline t.formats/pe/signature_cert]: oh, i'll remove it then with another commit.

**Pull request merged**